bookstore/_version.py

```diff
@@ -21,3 +21,23 @@
 version_info = (2, 3, 0, "", "dev")
 
 __version__ = ".".join(map(str, version_info[:3])) + ".".join(version_info[3:])
+
+
+def _check_version(bookstore_version, log):
```

Comment:
Maybe give the function a boolean return value that indicates if the request version is supported or not?